test/jdk/com/sun/jdi/JdwpAttachTest.java

```diff
@@ -168,6 +173,15 @@ private static void setConnectorArg(Map<String, Connector.Argument> args, String
         arg.setValue(value);
     }
 
+    // Attach to localhost tries to connect to both IPv4 and IPv6 loopback addresses.
+    // But sometimes it causes interference with other processes which can listen on the same port but different
+    // loopback address.
```

Comment:
Move a few words from the end of the 2nd line of the comment down to the 3rd.

Reply:
Done.